cmd/cl-adm/cmd/deploy/deploy_peer.go

```diff
@@ -84,6 +86,8 @@ func (o *PeerOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.Namespace, "start-namespace", app.SystemNamespace,
 		"Namespace where the ClusterLink components are deployed if --start is set.")
 	fs.StringVar(&o.Ingress, "start-ingress", string(apis.IngressTypeLoadBalancer),
+		"Represents the type of service used to expose the ClusterLink deployment (LoadBalancer/NodePort/none), if --start is set.")
+	fs.Uint16Var(&o.IngressPort, "start-ingress-port", 443, "Represents the ingress port."+
```

Comment:
if there are dependencies between flags (e.g., `start-ingress-port` only valid with `--start`) consider checking for those and reporting expectations failures (e.g., logging that ingress port is ignored when defined without start, logging that using default ingress port when start is defined but no flags...)

Reply:
Also, if using NodePort, there are probably checks you need to make (e.g., there's a specific range assigned in k8s for NodePort), or be ready to fail on creating the service and report back to the user.

Reply:
Done for checking the flags, about the nodeport there is a check before  the nodeport creation (see in pkg/bootstrap/platform/k8s.go in this commit)